MIPs/mip-4.md

```diff
@@ -0,0 +1,50 @@
+---
+mip: 4
```

Comment:
If you set it to "x" and change the filename to mip-x.md, then it should get an automatic number assignment when merged.
```suggestion
mip: x
```